UKTrains.indigoPlugin/Contents/Server Plugin/text2png_modern.py

```
@@ -0,0 +1,796 @@
+                if paren_depth == 0:
+                    stations.append(current_station.strip())
+                    current_station = ""
+
+        if current_station.strip():
+            stations.append(current_station.strip())
+
+        # Build lines that fit within width
+        current_line = ""
+        for station in stations:
+            test_line = f"{current_line} {station}".strip()
+            bbox = draw.textbbox((0, 0), f"› {test_line}", font=fonts['calling_points'])
```

Ruff (0.14.14)

[warning] String contains ambiguous `›` (SINGLE RIGHT-POINTING ANGLE QUOTATION MARK). Did you mean `>` (GREATER-THAN SIGN)?

(RUF001)